helium/src/scss/_sales_page.scss

```diff
@@ -2,7 +2,7 @@
 @import "ycthemelib/responsive";
 @import "constants";
 
-.sales-page {
+.course-sales-page {
```

Comment:
learn.js automatically adds a class corresponding to the page eg. in this case it will add `sales-page`